content/en/cloudprem/install/azure_aks.md

```diff
@@ -97,6 +105,13 @@ Before installing CloudPrem on AKS, ensure your Azure environment is properly co
    Here is an example of a `datadog-values.yaml` file with overrides for Azure:
 
    {{< code-block lang="yaml" filename="datadog-values.yaml">}}
+# Datadog configuration
+datadog:
+  # The Datadog [site](https://docs.datadoghq.com/getting_started/site/) to connect to. Defaults to `datadoghq.com`.
```

Comment:
Code blocks don't support hyperlinks, this will show up as the URL in the docs.

```suggestion
  # The Datadog site (https://docs.datadoghq.com/getting_started/site/) to connect to. Defaults to `datadoghq.com`.
```